src/main/java/fi/fmi/avi/converter/tac/bulletin/GenericMeteorologicalBulletinParser.java

```diff
@@ -88,158 +79,36 @@ public void setTACLexer(final AviMessageLexer lexer) {
 
         //Lex each the contained message again individually to collect more info:
         String msg;
-        Lexeme lm;
 
-        String bulletinID = null;
-        //GTSExchangeFileInfo bulletinMetadata = null;
+        GTSExchangeFileInfo bulletinMetadata = null;
         if (hints != null && hints.containsKey(ConversionHints.KEY_BULLETIN_ID)) {
-            bulletinID = hints.get(ConversionHints.KEY_BULLETIN_ID, String.class);
+            String bulletinID = hints.get(ConversionHints.KEY_BULLETIN_ID, String.class);
             //the bulletinID could be used to provide full (or more complete message time info:
-            /*
             try {
                 bulletinMetadata = GTSExchangeFileInfo.Builder.from(bulletinID).build();
             } catch (IllegalArgumentException iae) {
-                result.addIssue(new ConversionIssue(ConversionIssue.Severity.WARNING, ConversionIssue.Type.SYNTAX, "Could not parse bulletin metadata "
-                        + "from bulletinID '" + bulletinID + "'"));
+                result.addIssue(new ConversionIssue(ConversionIssue.Severity.WARNING, ConversionIssue.Type.SYNTAX,
+                        "Could not parse bulletin metadata " + "from bulletinID '" + bulletinID + "'"));
             }
```

Comment:
Is there a reason for this being uncommented? Right now I don't remember the actual reason for this being commented out, but I have a hunch it lead to some problems.

Reply:
Seems that I accidentally uncommented this section of code.